script/Arcueid/Day 12/12_00_ARC12_1B.txt

```diff
@@ -60,6 +60,8 @@ The physics teacher frowns, but his expression then softens somewhat.
 [sha:3bc5df80e77f3671406ba0d286151bbeebeb38eb]{
 -- Page 19, Offset 30532.
 -- そういったワケだから、放課後は茶道室に行くように」
+// group pr check: change all that to say
+// suggestions: With all that said, That is to say,
```

Comment:
That is to say,

Reply:
I prefer "that is to say" but "all that to say" is also fine by me.